addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/FreeboxAirPlayAudioSink.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.freebox.internal;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.audio.AudioFormat;
+import org.eclipse.smarthome.core.audio.AudioHTTPServer;
+import org.eclipse.smarthome.core.audio.AudioSink;
+import org.eclipse.smarthome.core.audio.AudioStream;
+import org.eclipse.smarthome.core.audio.FixedLengthAudioStream;
+import org.eclipse.smarthome.core.audio.URLAudioStream;
+import org.eclipse.smarthome.core.audio.UnsupportedAudioFormatException;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.util.ThingHandlerHelper;
+import org.openhab.binding.freebox.config.FreeboxAirPlayDeviceConfiguration;
+import org.openhab.binding.freebox.handler.FreeboxThingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This makes a AirPlay device to serve as an {@link AudioSink}-
+ *
+ * @author Laurent Garnier - Initial contribution for AudioSink and notifications
+ */
+public class FreeboxAirPlayAudioSink implements AudioSink {
+
+    private final Logger logger = LoggerFactory.getLogger(FreeboxAirPlayAudioSink.class);
+
+    private static final AudioFormat MP3_96 = new AudioFormat(AudioFormat.CONTAINER_NONE, AudioFormat.CODEC_MP3, null,
```

Comment:
Minor: with static imports and using the autoboxing of Integer, it could fit a single line?
```java
 private static final AudioFormat MP3_112 = new AudioFormat(CONTAINER_NONE, CODEC_MP3, null, null, 112000, null);
```